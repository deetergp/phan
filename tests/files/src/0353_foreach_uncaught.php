<?php
function testbadloop() {
    foreach ($$y as $$x) {
    }
}
