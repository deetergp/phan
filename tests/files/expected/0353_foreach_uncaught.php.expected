%s:3 PhanUndeclaredVariable Variable $x is undeclared
%s:3 PhanUndeclaredVariable Variable $y is undeclared
